### DISCUSSION TITLE
Enable Windows Plays in Playbook

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#7274] [Unsupported Versions] [answered]
Hello!

I have two basic Playbook questions:

1. Is it safe to just enable all Windows Plays in Playbook or do I overwhelm my distributed deploament?
2. How can I enable several Plays at once in Playbook? I can select multiple Plays, but was not able to find a way to change their status.

Thanks in advance!

Cheers, Ben

**Comment by dougburks** [comment, marked as answer]
> 1\. Is it safe to just enable all Windows Plays in Playbook or do I overwhelm my distributed deploament?

As with most detection technologies, you'll want to avoid blindly enabling all detections. In particular, from https://docs.securityonion.net/en/2.3/playbook.html#putting-a-play-into-production:
![image](https://user-images.githubusercontent.com/1659467/155532328-a047542f-64f8-48e6-b6fa-f13f25ac9978.png)

> 2\. How can I enable several Plays at once in Playbook? I can select multiple Plays, but was not able to find a way to change their status.

Select multiple plays then click the ellipsis (...) in the rightmost column for one of them and click Status. Whatever status you choose will apply to all selected plays.
![image](https://user-images.githubusercontent.com/1659467/155532660-fe99f55f-8ae4-402e-a2d3-1f59274296b5.png)

**Reply by ben-sec** [reply]
Thanks Doug!
